lightwallet-client-lib/src/main/java/co/electriccoin/lightwallet/client/util/Disposable.kt

```diff
@@ -0,0 +1,42 @@
+package co.electriccoin.lightwallet.client.util
+
+import java.io.IOException
+
+/**
+ * A [Disposable] is a source or destination of data that can be closed/disposed.
```

Comment:
A `TorWalletClient` is not just a stream / a source or destination of data; it is an object for interacting with a remote service. That implies `Disposable` needs to be implementable for more than this:
```suggestion
 * A [Disposable] is an object that can be closed/disposed.
```
(or whatever substitute for "object" makes sense in Kotlin).